docs/topics/native/native-get-started.md

```diff
@@ -48,30 +74,27 @@ After the first run, you will see the corresponding run configuration on the top
 
```

Comment:
WDYT about changing the future tense on lines 69 and 73 (from the modified file) to present tense as well?